src/recursive_sorting/recursive_sorting.py

```diff
@@ -6,13 +8,49 @@ def merge( arrA, arrB ):
     
     return merged_arr
 
+# 1. While your data set contains more than one item, split it in half
+# 2. Once you have gotten down to a single element, you have also *sorted* that element 
+#    (a single element cannot be "out of order")
+# 3. Start merging your single lists of one element together into larger, sorted sets
+# 4. Repeat step 3 until the entire data set has been reassembled
 
 # TO-DO: implement the Merge Sort function below USING RECURSION
 def merge_sort( arr ):
     # TO-DO
+    if len(arr) > 1:
+        middle = math.floor(len(arr)/2)
+        left = arr[:middle]
+        right = arr[middle:]
+
+        merge_sort(left)
+        merge_sort(right)
+
+        index_1 = 0
+        index_2 = 0
+        index_3 = 0
+
+        while index_1 < len(left) and index_2 < len(right):
+            if left[index_1] < right[index_2]:
+                arr[index_3] = left[index_1]
+                index_1 += 1
+            else:
+                arr[index_3] = right[index_2]
+                index_2 += 1
+            index_3 += 1
+
+        while index_1 < len(left):
+            arr[index_3] = left[index_1]
+            index_1 += 1
+            index_3 += 1
 
```

Comment:
Nice work. I appreciate that you didn't just use Brady's code and call it a day!